src/explore-education-statistics-admin/src/components/editable/__tests__/Comments.test.tsx

```diff
@@ -67,6 +67,7 @@ describe('Comments', () => {
             },
           ]}
           onChange={noop}
+          onToggle={noop}
```

Comment:
If you make `onToggle` optional, you don't need to add it for all of these test cases.

Reply:
I spoke with Mark about this and he told me to add it to the test cases

Reply:
I'd err the other way as it's not actually mandatory to make the component work correctly in the context of its core functionality. 

The component is perfectly able to function without an `onToggle`, as exhibited by the tests. It's also annoying to have to add an empty default for any new test cases as well.